Sources/Basics/Triple+Basics.swift

```diff
@@ -148,6 +148,10 @@ extension Triple {
     }
 
     public var executableExtension: String {
+        guard ![.wasm32, .wasm64].contains(self.arch) else {
```

Comment:
How about adding `isWasm` like llvm's one?: https://github.com/llvm/llvm-project/blob/031f9f331723e6bebc405ffdee4b8a87a5fc0472/llvm/include/llvm/TargetParser/Triple.h#L1005

I think `[.wasm32, .wasm64].contains(self.triple.arch)` appears repeatedly, so it's worth adding memory space size independent predicate method.

Reply:
Good point, I've added it on an extension in `Basics/Triple+Basics.swift`